src/constructors/css.js

```diff
@@ -1,4 +1,5 @@
 import camelize from 'fbjs/lib/camelizeStyleName'
+import isObject from "lodash/isObject"
```

Comment:
Should this be something about "simple object" or "flat object" if you're just iterating over keys & values? That way we don't end up setting `font-size` to `[object Object]` :)